src/tree-selector/index.ts

```diff
@@ -0,0 +1,6 @@
+import TreeSelector from './treeSelectorComponent';
+import * as angular from 'angular';
+module treeSelector {
```

Comment:
I know I'm strict, but you have some many extra newlines, move one here between the `import` and the `module`, so my :eyes: will rest more while reading :wink: